src/clients/java/samples/two-phase/src/main/java/Main.java

```diff
@@ -67,7 +67,6 @@ public static void main(String[] args)
 	    assert accounts.getCapacity() == 2;
 
 	    while (accounts.next()) {
-		accounts.getTimestamp();
```

Comment:
@batiati I don't know why these `.getTimestamp()` calls were in here in the first place. But they were in many places.

Reply:
They do not seem to be necessary. So I got rid of them. Mentioning it in case they are actually necessary.